app/src/main/res/layout/fragment_onboarding_icons.xml

```diff
@@ -23,10 +23,11 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="4dp"
+        android:autoSizeTextType="uniform"
```

Comment:
Does the `autoSizeTextType="uniform"`, which you added to all text views, actually change something? I think it would only adjust the font size if the text view size is fixed.